services/avatar.service.js

```diff
@@ -0,0 +1,48 @@
+const AvatarGenerator = require("avatar-generator");
+const path = require("path");
+const fs = require("fs-extra");
+const User = require("../api/users/user/user.model");
+
+const generate = async (id) => {
+  const avatar = new AvatarGenerator({
+    parts: ["background", "face", "clothes", "head", "hair", "eye", "mouth"],
+    partsLocation: path.join(__dirname, "../img"),
+    imageExtension: ".png",
+  });
+  const variant = "male";
+  const image = await avatar.generate("email@example.com", variant);
+  image.png().toFile(`tmp/${id}.png`);
```

Comment:
move `tmp/${id}.png` to config

export const youFunctionName = (id ) => `tmp/${id}.png`

Reply:
moved to created config service